rollback1.py

```diff
@@ -183,29 +271,39 @@ def edit_sign(self, name):
         valueE += "\" ,  \"title\": \"" + name
         valueE += "\", \"params\":[{\"name\": \"test\", \"text\": \"test\"}]}"
         dataEdit = {"Id": funcs.get_count(url, "signatures", token),
-                   "Value": valueE,
-                   "Conditions": "true"}
+                    "Value": valueE,
+                    "Conditions": "true"}
         res = self.call("EditSign", dataEdit)
-        
+
     def test_rollback1(self):
-        global url, prKey, token, waitTx
-        self.conf = config.readMainConfig()
-        url = self.conf["url"]
-        prKey = self.conf['private_key']
-        host = self.conf["dbHost"]
-        db = self.conf["dbName"]
-        login = self.conf["login"]
-        pas = self.conf["pass"]
-        waitTx = self.conf["time_wait_tx_in_block"]
+        self.addNotification()
+        self.addBinary()
+        tableName = self.addUserTable()
+        self.insertToUserTable(tableName)
+        # Save to file block id for rollback
+        RollbackBlockId = funcs.get_max_block_id(url, token)
+        file = os.path.join(os.getcwd(), "blockId.txt")
+        with open(file, 'w') as f:
+            f.write(str(RollbackBlockId))
+        # Save to file user table name
+        tableNameWithPrefix = "1_" + tableName
+        file = os.path.join(os.getcwd(), "userTableName.txt")
+        with open(file, 'w') as f:
+            f.write(str(tableNameWithPrefix))
+        # Save to file user table state
+        dbUserTableInfo = utils.getUserTableState(host, db, login, pas, tableNameWithPrefix)
```

Comment:
same

Reply:
used camelCase similar to the already existing variables